tests/unit/dojson/test_dojson_hep.py

```diff
@@ -1630,4 +1716,78 @@ def test_mashed_publication_info_from_773():
     }
     result = hep.do(create_record(snippet))
 
+    assert validate(result['publication_info'], subschema) is None
     assert expected == result['publication_info'][0]
+
+    expected = [
+        {
+            'c': ['45'],
+            'p': 'IAU Symp.',
+            '0': 1408366,
+            'w': 'C08-06-09',
+            'v': '354',
+            'y': 2008
+        }
+    ]
+    result = hep2marc.do(result)
+
+    assert expected == result['773']
+
+
+def test_isbns_from_020_a_only():
+    schema = load_schema('hep')
+    subschema = schema['properties']['isbns']
+
+    snippet = (
+        '<datafield tag="020" ind1=" " ind2=" ">'
+        '  <subfield code="a">9780198759713</subfield>'
+        '</datafield>'
+    )  # record/1510325/export/xm
```

Comment:
`/export/xm` is implied by the fact that this is MARCXML, so it can be dropped here.